iothub/device/src/Transport/Mqtt/MqttIotHubAdapter.cs

```diff
@@ -82,16 +82,18 @@ private enum StateFlags
         private DateTime _lastChannelActivityTime;
         private StateFlags _stateFlags;
 
-        private ConcurrentDictionary<int, TaskCompletionSource> _subscribeCompletions = new ConcurrentDictionary<int, TaskCompletionSource>();
-        private ConcurrentDictionary<int, TaskCompletionSource> _unsubscribeCompletions = new ConcurrentDictionary<int, TaskCompletionSource>();
+        private readonly ConcurrentDictionary<int, TaskCompletionSource> _subscribeCompletions = new ConcurrentDictionary<int, TaskCompletionSource>();
+        private readonly ConcurrentDictionary<int, TaskCompletionSource> _unsubscribeCompletions = new ConcurrentDictionary<int, TaskCompletionSource>();
 
         private int InboundBacklogSize => _deviceBoundOneWayProcessor.BacklogSize + _deviceBoundTwoWayProcessor.BacklogSize;
 
         private readonly ProductInfo _productInfo;
         private readonly ClientOptions _options;
 
-        private ushort _packetId = 0;
-        private SpinLock _packetIdLock = new SpinLock();
+        // default value for ushort is 0.
+        private ushort _packetId;
+
+        private SpinLock _packetIdLock;
```

Comment:
is this initialization removal intentional?

Reply:
yes, this is a struct and it automatically gets initialized with all its properties (default value for all properties)